doc/Learn/HowTo-UseThemeService.md

```diff
@@ -0,0 +1,56 @@
+---
+uid: Uno.Extensions.ThemeService
+---
+
+# How to use Theme Service
+
+The Uno Extensions library includes a `ThemeService` that facilitates dynamic theme switching across your Uno Platform applications. This service is especially useful for applications requiring runtime theme changes and for persisting user theme preferences.
```

Comment:
This topic explains how to use the `ThemeService` for runtime theme switching and persisting user theme preferences.